boards/arm/particle_argon/doc/particle_argon.rst

```diff
@@ -0,0 +1,144 @@
+.. _particle_argon:
+
+Particle Argon
+##############
+
+Overview
+********
+
+The Particle Argon is a Wi-Fi enabled development board with a Nordic
+Semiconductor nRF52840 for mesh support and an ESP32 for Wi-Fi.  The
+board was developed by Particle Industries and has a SWD connector on it
+for programming.
+
+It is equipped with a onboard LIPO circuit and conforms to the
+Adafruit Feather formfactor.
+
+Zephyr applications use the particle argon board configuration
+to run on the particle argon board hardware. It provides
+support for the Nordic Semiconductor nRF52840 ARM |reg| Cortex |reg|-M4F SoC
+with an integrated 2.4 GHz transceiver supporting Bluetooth |reg| Low Energy
+and IEEE |reg| 802.15.4.
+
+For more information about the Particle Argon board:
+
+- `Argon Datasheet`_
+- `Argon Hardware Files`_
+
+Hardware
+********
+
+On the front of the board are RGB-LED, LED and LIPO circuitry.
+The RGB-LED is controlled by the nRF52840 via GPIO pins.
+
+.. figure:: img/argon-top.png
+     :width: 150px
+     :align: center
+     :alt: Particle Argon
+
+     Particle Argon (Credit: Particle Industries)
+
+Power supply
+============
+
+The board is optimized for low power applications and supports two
+power source configurations: battery and micro USB connector.
+
+It contains circuitry for LIPO usage and can be charged via the USB port.
+
+Supported Features
+==================
+
+The particle_argon board configuration supports the following
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| NVIC      | on-chip    | nested vectored      |
+|           |            | interrupt controller |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial port          |
++-----------+------------+----------------------+
+| I2C       | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| SPI       | on-chip    | spi                  |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth            |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+
+Connections and IOs
+===================
+
+Please see the `Argon Datasheet`_ for more information.
+
+Adapter LEDs
+------------
+
++-------+-----------------------+--------------------------------+
+| Name  | Type                  | Usage                          |
```

Comment:
Do we need this third column (Usage) if all the items are a dash?

Reply:
I don't think we need it as a table at all.  I'll switch this section to use lists instead of a table; seems more common.

Reply:
replaced this section in all three boards with text that provides more clear indication of peripheral assignments.